src/custom/hooks/web3.ts

```diff
@@ -125,6 +126,25 @@ export function useEagerConnect() {
     return () => timeout && clearTimeout(timeout)
   }, [active])
 
+  useEffect(() => {
+    // fix for this https://github.com/gnosis/cowswap/issues/1923
+    // check if current connector is of type WalletConnect
```

Comment:
Again, as a reader, i prefer to read first what is the whole thing about, and then if i want more details i can read the issue


Also, u should start in capital letter the comments, also use the imperative form. `Check if current`

Reply:
Done